src/NIScopeDigitalMeasurement/README.md

```diff
@@ -1,29 +1,33 @@
 ## NI-SCOPE Digital Measurement 
 
-This example performs measurement for digital signals using NI SCOPE.
+This measurement reads a digital signal and analyzes its characteristics (High State Level, Frequency, High Time, Duty Cycle, etc.) using NI-SCOPE.
 
 
 ### Features
 
 - Uses the NI-SCOPE LabVIEW API.
 - Pin-aware, supporting one session and one pin.
   - Uses the same selected measurement function and range for all selected pin/site combinations.
+  - An example pin map file is included for this measurement. However, you'll need to setup your own pin map file according to your own system.
 - Includes InstrumentStudio project files.
 - Includes a TestStand sequence showing how to configure the pin map, register instrument sessions with the session management service, and run a measurement.
   - For the sake of simplicity, the TestStand sequence handles pin map and session registration and unregistration in the `Setup` and `Cleanup` sections of the main sequence. For **Sequential** and **Batch** process model use cases, these steps should be moved to the `ProcessSetup` and `ProcessCleanup` callbacks.
 - Uses the NI gRPC Device Server to allow sharing instrument sessions with other measurement services when running measurements from TestStand.
 
-### Required Driver Software
+### Run a Measurement
+In order to use a measurement, you will need to start the measurement service and open the measurement plugin in InstrumentStudio at first, refer to the main repo [Readme](TBD) for more details.
+- Select the expected Pin in your system .
+- Setup the measurement vertical range, offset and coupling settings.
+- Setup the measurement timing and trigger settings.
+- Click the Run button. The measured value will be displayed in the measurement indicators and waveform.
```

Comment:
can you add the surrounding apostrophes around Run like `Run` ? Just to match how we write it in the Main readme.